### DISCUSSION TITLE
Acteur Bouton Poussoir sur synapps

### DISCUSSION THREAD

**Discussion by JeremieDENAEGEL** [witsa/synapps#581] [Q&A]
Bonjour,

Est-ce-que quelqu'un sait comment reproduire un bouton poussoir avec les acteurs SYNAPPS ? 
Afin de faire la même action que la représentation "bouton poussoir" des acteurs Objet sur synoptique

Merci d'avance
Jérémie

**Comment by WITNicolas** [comment]
Salut
Tu trouveras ton bonheur ici :)
https://github.com/witsa/synapps/discussions/558
